content/en/tracing/trace_collection/open_standards/otel_collector_datadog_exporter.md

```diff
@@ -313,7 +308,7 @@ To use the OpenTelemetry Operator:
    spec:
      mode: daemonset
      hostNetwork: true
-     image: otel/opentelemetry-collector-contrib:0.59.0
+     image: otel/opentelemetry-collector-contrib
```

Comment:
I assume this then defaults to latest, based on the description of the PR.

Reply:
Yup!